jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/delegate/AbstractFieldDelegate.java

```diff
@@ -234,8 +234,6 @@ public void setInvalid(boolean invalid) {
 
     public void setConversionInvalid(boolean conversionInvalid) {
         this.conversionInvalid = conversionInvalid;
-
-        updateInvalidState();
```

Comment:
check that https://github.com/jmix-framework/jmix/issues/1456 still works

Reply:
Missed one case from #1456 the issue. Now it is fixed